packages/vuetify/build/webpack.prod.config.js

```diff
@@ -1,5 +1,4 @@
 const merge = require('webpack-merge')
-const HappyPack = require('happypack')
```

Comment:
Why'd you remove happypack? It should so be removed from dependencies if there's a good reason not to use it any more.